backstop/tests/carousel.html

```diff
@@ -0,0 +1,355 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>Carousel | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      section[id^='demo'] {
+        width: auto;
+        margin-bottom: 22px;
+      }
+
+      .demo-columns {
+        display: flex;
+        width: 100%;
+        height: 100%;
+        background-color: var(--iui-color-background-4);
+      }
+
+      .demo-column {
+        width: 50%;
+        height: 100%;
+        box-sizing: border-box;
+      }
+
+      .demo-left {
+        padding: 11px 12px;
+      }
+
+      .demo-right {
+        background-image: linear-gradient(to right, var(--iui-color-background-4), rgba(var(--iui-color-background-4-rgb), 0)), url('../assets/image-test.jpg');
+        background-position: center;
+        background-size: cover;
+      }
+    </style>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/chevron-left"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/chevron-right"
+    ></script>
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>Carousel</h1>
+    <hr />
+
+    <h2>Default</h2>
+    <section id="demo-default">
+      <div
```

Comment:
I would suggest wrapping carousel in some smaller div.. Cause now seeing it full screen is a little bit too big and navigation is not even seenable

Reply:
I applied `.iui-carousel { max-width: 600px; }` in the demo's `<head>` styling, is that ok?

Reply:
looks much better.

Reply:
Just maybe isntead we could have a custom class or even apply through style not to confuse, which part is coming from our css, which is just for demo purposes?